### PR TITLE
Fix Bootstrap payload for esxi fails when IPv4 is set as static

### PR DESCRIPTION
**Pull request by keedya**
@RackHD/corecommitters @brianparry @srinia6 @dalebremner @geoff-reid @BillyAbildgaard

### CODE REVIEW

File: data/templates/esx-ks [hunk 1]
```diff
@@ -138,7 +138,7 @@ cp /var/log/esxi_install.log "/vmfs/volumes/datastore1/firstboot-esxi_install.lo
     esxcli network vswitch standard add -v "<%=n.switchName%>"
     <% if( undefined !== n.uplinks ) { %>
       <% n.uplinks.forEach(function(s) { %>
-        currdev = <%=s%>
+        currdev=<%=s%>
```
Comment (yyscamper) [inline data/templates/esx-ks]: I think the line 158 still need the same fix.

**Comment by yyscamper** [comment]
:+1: after fix my comment.

File: data/templates/esx-ks (hunk 1 above)
Comment (keedya) [inline data/templates/esx-ks]: @yyscamper Thanks Felix

**Comment by keedya** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Comment by dalebremner** [comment]
+1

**Comment by brianparry** [comment]
:+1:

**Pull request merged**